code/modules/spells/spell.dm

```diff
@@ -33,7 +33,7 @@ GLOBAL_LIST_INIT(spells, typesof(/obj/effect/proc_holder/spell)) //needed for th
 		return
 	if(user.ranged_ability && user.ranged_ability != src)
 		if(forced)
-			to_chat(user, "<span class='warning'><b>[user.ranged_ability.name]</b> has been replaced by <b>[name]</b>.")
```

Comment:
wait, how'd you miss this one and get all the others?

Reply:
It looked daunting with the long name... Should it be
"to_chat(user, "<span class='warning'><b>[user.ranged_ability]</b> has been replaced by <b>[name]</b>.")"?

Reply:
nah, it's fine as-is